### DISCUSSION TITLE
I'm an active lazer user and I think I dislike the new visual changes to lazer.

### DISCUSSION THREAD

**Discussion by snalgae** [ppy/osu#28919] [Q&A] [answered]
I don't know how to describe this. I think I just got used to the way lazer looked before the update.

I dislike that the beatmaps are now overlapping it feels like they're not fitting neatly together so they have to stack a little. I do not remember ever clicking on the space between them and I do not find a problem of having them.
I dislike the new colors for timeline points in the editor (i associate timing points with yellow and kiai times with purple and having them changed to match stable makes me confused). 
I liked the kiai time with distinct borders and semi-transparency inside, now it's just a blob (maybe break time should be different, now that something I liked changed), and ending tick of 1x scroll speed that is created from ending kiai time is weird. 
I don't like that the timing points are semi-transparent (i assume they're changed to be less important, but they now feel invisible). 
I can see that the bpm text on red timing point arrow just doesn't fit. 
I find the new beat snap ticks to be too big and goofy, I liked seeing different thicknesses for the ticks to show how smaller the division is, but now they feel the same. (I mean they have different colors and still decrease in thickness, but barely compared to their overall thickness now, so it just looks janky. I think ticks' thickness should've been scaled instead of a flat increase by a certain amount).
And the fact that the height changes only slightly makes me think that this is just a transition point and all timing ticks are supposed to be the same thing at the end.

I think the smaller timeline is fine, but I had no complaints about it, I didn't really pay attention to the size of the editor (i don't find the editor claustrophobic, but maybe because I use a widescreen monitor)
I do find the asymmetry of having the top timing ticks exposed useful for readability.
I do not like the distortions
I think the customise mods tab should be next to the search bar, because I usually pick my mods from the left side of the screen and having to move my cursor to move

I understand that the changes are meant for the stable users and it's all just integrating stable features to not alienate stable players (which are a majority) with having things different than they're used to. I feel like I'm alienated that these visual changes are meant to make lazer feel more like stable (at least that's what I think is happening? I'm not confident). 
The visual changes also feel like hacks. (Let's change the colors for these to ones that match stable! Let's add some points to the thickness of the ticks and increase them in size with some off-setting! Let's swap the kiai model to one that break uses!) I mean basically everything is, but these feel more hacky, like there's a need to meet some quota on stuff to show in the changelog to excite the userbase and the stuff that's been developed is more background-based and is in wip. I mean there was a streak of news arriving daily before the changelog, why I can't assume that these news were meant to get stable users to follow the osu news and get to know lazer changes? (this sounds like a conspiracy theory xd)
I think this is like the "This is different and I hate it", but from the lazer-user side. I do feel frustrated and I find it important to acknowledge it. I also find it important to acknowledge that the osu! team is incredibly busy and hard-working and this ostentatious rant can be seen as insensitive and emotional. It is. But I think this still has feedback, but rather biased and in need of filtering. I think I want to come back to this in a few days-week to have a clearer perspective on this.

Obviously satisfying every person's desire for how editor functions is impossible. I wonder if it makes more sense to have an ability to create editor themselves to satisfy a person's needs, but that feels like something that is too niche to spend a lot of time on, which is unpopular to develop and there's already a direction for this game I would so assume, so I assume this project is highly unlikely to be developed.

**Comment by peppy** [comment, marked as answer]
thanks for you opinion

**Reply by snalgae** [reply]
i'm open to provide information on some details if you need so

**Reply by peppy** [reply]
We consider each opinion but your opinion is just your own and doesn't necessarily align with ours.

There's nothing to immediately action on and nothing further to provide at this stage.

**Reply by snalgae** [reply]
okay thank you for answering anyways

**Comment by bdach** [comment]
> like there's a need to meet some quota on stuff to show in the changelog to excite the userbase and the stuff that's been developed is more background-based and is in wip

i take personal offense to this because none of this was my whims. all of it was feedback from stable editor users, given as rationale as why they don't want to switch. i do not do change for the sake of change.

> I think this is like the "This is different and I hate it", but from the lazer-user side.

so what, do i revert the changes and get "this is different and i hate it" _from stable users_? do you see how this is an unwinnable position?

**Reply by snalgae** [reply]
I do not know how to answer to this except to apologize for my post. I should have withheld this opinion and processed it before uploading it.
> i take personal offense to this because none of this was my whims. all of it was feedback from stable editor users, given as rationale as why they don't want to switch. i do not do change for the sake of change.

I like to negatively speculate about something I dislike as a reason why I disagree with it. This is a pattern I start to notice and I should address it.

> so what, do i revert the changes and get "this is different and i hate it" from stable users? do you see how this is an unwinnable position?

That's why I pointed this out and I wrote that accommodating for everyone is impossible at the end of my post, though in an obtuse way. I really did not think what I was writing because at the time I felt that it was important to show my hatred. That's kinda awful now that I look at it.

I feel calmer about this and while I still have some things I'm unhappy with (colors in the timeline, imperceptible thickness differences between ticks), I'm now okay with most of the changes I complained about.

**Reply by Tenexxt** [reply]
it would be a hell to make, but you could make this changable in the settings. like, old assets are for a skin for song select and future ui changes, and all new changes are on a new skin. As for editor colours, they could be changed in the settings also. this would probably take time to make, but its an option that would satisfy a lot, especialy the first part of this reply.

**Reply by snalgae** [reply]
i've read in the discord that this is the scenario they want to avoid

**Reply by Tenexxt** [reply]
> i've read in the discord that this is the scenario they want to avoid

If you don't mind me asking, why? Did the devs gave a reason for that, or is that currently unexplained?

**Reply by bdach** [reply]
to quote a comment of mine from elsewhere

> the vicious cycle:
>
> - do something using common sense without looking at stable
> - users: "wtf make this like stable"
> - make it like stable instead to make users happy 
> - users: "wtf why did you take away the new behaviour why is this not a toggle"
>
> rinse and repeat for every. little. thing. you may not care about "code maintainability" and say that this is on us to get right, but i personally do not want to *ever* read a bug report that says "when you enable toggle 543, 753, and 1171, the game breaks".
>
> every toggle that gets added is a potential rake for us to step on later. wanna do a new feature? you better enumerate *every single toggle* that may interact incorrectly with this new feature, and make sure the feature behaves sensibly with every one of them!
>
> do you see why "just make it a toggle" is not something i respond well to?

**Reply by Tenexxt** [reply]
Yes, thanks for the clarification

**Comment by ChiNoel-osu** [comment]
I do feel the overlapping beatmaps are little bit cramped but meh it's not unacceptable

**Reply by Squiddu** [reply]
that side is gonna get a visual refresh anyways and this is how stable was so its fine
